lib/util/QueuePool.js

```diff
@@ -68,11 +72,13 @@ class QueuePool {
                 p: queue.enqueue(req.id, req.item)
             });
         }
+        this._overflowCount++;
```

Comment:
Is this variable meant to be counting the size of the overflow queue (aka "number of items waiting to be inserted into a queue.")? If so, why isn't this incremented when `this.overflow.enqueue` is called? At present, this will only be incremented "when a request is at the front of the overflow queue", which isn't the same thing.